### PR TITLE
More gui

### PR DESCRIPTION
**Pull request by sstone93**

### CODE REVIEW

File: src/view/View.java [hunk 1]
```diff
@@ -40,7 +42,28 @@ public View(Controller c){
...
 	public void update(){
-		instituteLabel.setText("Enter password for: " + controller.getMainModel().getCurrentType());
+		MainModel model = controller.getMainModel();
+		
+		if(model.getCurrentMode() == MainModel.MODE.PASSWORD_ENTERED){
+			if(model.getCurrentStatus() == MainModel.LOGIN_STATUS.SUCCESS){
+				JOptionPane.showMessageDialog(this, "Correct password entered.", "Correct", JOptionPane.PLAIN_MESSAGE);
+			}
+			else {
+				JOptionPane.showMessageDialog(this, "Incorrect password entered.", "Incorrct", JOptionPane.PLAIN_MESSAGE);
```
Comment (ChantalForget) [inline src/view/View.java:52]: `Incorrect`

File: src/view/PasswordPanel.java [hunk 2]
```diff
@@ -145,7 +167,14 @@ public void actionPerformed(ActionEvent e){
 			}
 		});
 		showCheckBox.setBounds(600, 330, 150, 30);
+		showCheckBox.setSelected(true);
 		add(showCheckBox);
+		
+		clicksLabel = new JLabel("");
+
+		clicksLabel.setFont (clicksLabel.getFont ().deriveFont (24.0f));
```
Comment (ChantalForget) [inline src/view/PasswordPanel.java:175]: weird spacing for `getFont  () ` (are you allowed to do that?)

File: src/view/PasswordPanel.java [hunk 3]
```diff
@@ -53,6 +56,26 @@ public void focusGained(FocusEvent e){
...
+			}
+
+			@Override
+			public void removeUpdate(DocumentEvent e) {
+				callController();
+			}
+			
+			public void callController(){
+				String t = new String(entryTextField.getPassword());
+				controller.handleTextEnter(t);
```
Comment (ChantalForget) [inline src/view/PasswordPanel.java:77]: is this already made?

File: src/view/PasswordPanel.java (hunk 2 above)
Comment (sstone93) [reply src/view/PasswordPanel.java:175]: stop judging me... 😯

**Pull request merged**